src/pkg/cli/compose/validation.go

```diff
@@ -50,14 +50,9 @@ func ValidateProject(project *composeTypes.Project) error {
 }
 
 func validateService(svccfg *composeTypes.ServiceConfig, project *composeTypes.Project) error {
-	normalized := NormalizeServiceName(svccfg.Name)
-	if !pkg.IsValidServiceName(normalized) {
-		// FIXME: this is too strict; we should allow more characters like underscores and dots
+	if !pkg.IsValidServiceName(svccfg.Name) {
```

Comment:
Do we still need this? Going forward, we'll do any resource-specific normalization in CD anyway.